### PR TITLE
Fixed-size H* use, bigint

### PR DESCRIPTION
**Pull request by lexfrl**
Tests refactoring to satisfy https://github.com/NikVolf/pwasm-std/pull/10

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -4,7 +4,7 @@ version = "0.1.0"
 authors = ["NikVolf <nikvolf@gmail.com>"]
 
 [dependencies]
-pwasm-std = { git = "https://github.com/nikvolf/pwasm-std" }
+pwasm-std = { git = "https://github.com/nikvolf/pwasm-std", branch = "store_hash" }
```
Comment (NikVolf) [inline Cargo.toml]: can reference master now

**Comment by NikVolf** [comment]
lgtm!

File: src/logger.rs [hunk 2]
```diff
@@ -4,22 +4,24 @@
...
+use pwasm_std::bigint::U256;
+use pwasm_std::hash::H256;
 
-fn set_key_from_addr(key: u32, val: &[u8; 20]) {
+fn set_key_from_addr(key: u32, val: &[u8]) {
 	let mut full_key = [0u8; 32];
 	let mut full_val = [0u8; 32];
 
 	write_u32(&mut full_key[0..4], key);
 	full_val[12..32].copy_from_slice(val);
```
Comment (rphmeier) [inline src/logger.rs:15]: will panic now if val length isn't 20

Comment (NikVolf) [reply src/logger.rs:15]: Not a big deal, it's internal function used in limited number of cases
And we handle such panics inside wasm in a controlled way

**Pull request merged**